readme.md

```diff
@@ -1,54 +1,180 @@
 # Functional Programming Jargon
 
-Functional programming (FP) provides a lot of advantages and its popularity has been increasing as a result.  However each programming paradigm comes with its own unique jargon and FP is no exception.  By providing a glossary we hope to make learning FP easier.
+Functional programming (FP) provides many advantages, and its popularity has been increasing as a result. However, each programming paradigm comes with its own unique jargon and FP is no exception. By providing a glossary, we hope to make learning FP easier.
 
-JavaScript is popular language that is well suited to FP; especially with revision ES2015.  Therefore we have chosen to use JavaScript (ES2015) for our examples, and in some cases definitions.  This should make this glossary as accessible and as possible and introduce features from the new revision.
+JavaScript is popular language that is well suited to FP, especially with the ES2015 update.  Therefore, we have chosen to use JavaScript (ES2015) for our examples and, in some cases, definitions. By doing so, we hope to not only make this glossary as accessible as possible but also to introduce features from the new revision.
 
 *This is a [WIP](https://github.com/hemanth/functional-programming-jargon/issues/20); please feel free to send a PR ;)*
 
 Where applicable, this document uses terms defined in the [Fantasy Land spec](https://github.com/fantasyland/fantasy-land)
 
+## Functional programming libraries and projects for JavaScript
+
+Practical examples of popular and production-ready FP libraries, FP-inspired projects, and other learning resources:
+
+* [ramda](https://github.com/ramda/ramda)
+* [lodash](https://github.com/lodash/lodash)
```

Comment:
Why? I'm not sure I follow. Also, I'm splitting up my pull requests now. Should I wait for each merge to master before making further pull requests? Or make them all at once? I don't want to create merge conflicts on account of my own edits!